### DISCUSSION TITLE
seeking general feedback for copying from one image to another in distorted way (see image)

### DISCUSSION THREAD

**Discussion by owntheweb-archive** [ImageMagick/ImageMagick#4774] [Help] [answered]
Hello gurus. I'm pretty sure the answer is already out there, yet I'm having a hard time locating the words for the best google search. Here's an image to describe. :)

![how-to-distorted-copy](https://user-images.githubusercontent.com/611860/151413460-3f13b85a-352f-4a86-83d7-91f77708ffc6.png)

How do I copy an image in a distorted way to a revised distorted way like this? I don't need a specific answer, yet general info for cli use would be great.

Thanks for taking a look in advance!

**Comment by fmw42** [comment, marked as answer]
Use 4 point perspective warping. See https://legacy.imagemagick.org/Usage/distorts/#perspective

**Reply by owntheweb-archive** [reply]
Perfect! Thank you @fmw42.
